decko-rails/spec/controllers/card_controller_spec.rb

```diff
@@ -5,8 +5,13 @@
   include Card::Location
   include Capybara::DSL
   describe "- route generation" do
+    routes { Decko::Engine.routes }
 
     it "should recognize type" do
+      all_routes = Rails.application.routes.routes
+
+   require 'rails/application/route_inspector'
```

Comment:
I should at least comment out the require.  This displays the routs much like rake routes.  Oddly, it displays all the routes even when it won't match tem because of the missing routes block as above.